scripts/boxstarter.ps1

```diff
@@ -1,7 +1,12 @@
+$OS = Get-WmiObject -Class win32_OperatingSystem -namespace "root\CIMV2"
 $WinlogonPath = "HKLM:\Software\Microsoft\Windows NT\CurrentVersion\Winlogon"
 Remove-ItemProperty -Path $WinlogonPath -Name AutoAdminLogon
 Remove-ItemProperty -Path $WinlogonPath -Name DefaultUserName
 
+if ($OS.Version -eq "6.1.7601") {
+    C:\Windows\System32\schtasks.exe /Delete /TN "setup_boxstarter" /f
```

Comment:
I dont think you need a scheduled task. Inside packer the packer run, the scripts are run as a local account.